src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+class Ball {
+    private final Colors color;
```

Comment:
Because in any place of usage it's confusing when single value of enum has type name is in multiple form
```suggestion
    private final Color color;
```